docs/UserGuide.md

```diff
@@ -236,6 +236,45 @@ Perhaps for some reason, you decide not to go through with an application. Maybe
 
 :information_source: Note the successful command execution message in the system feedback box upon successful deletion of the company.
 
+## Use Case #3: Editing a company or a role <a id="usecase3"></a>
+Say you received an email for an interview and the point of contact's email is different.
+Here's how you can make changes to the application:
+1. [Edit the company](#c-edit-c) from the company list.  
+![EditCompanyDemo](images/EditCompanyDemo.png)  
+:information_source: Note the successful command execution message in the system feedback box upon successful edit of the company.
+
+Also, you can change the status of your internship application and set the reminder for the interview date by making this entry:
+2. [Edit the role](#c-edit-r) from a given company.  
+![EditRoleDemo](images/EditRoleDemo.png)  
+:information_source: Note the successful command execution message in the system feedback box upon successful edit of the role.  
+:information_source: Note the change in color of the [role tag](#tags).
+
+## Use Case #4: Setting reminder window for reminder list <a id="usecase4"></a>
+Upon next opening of Tinner from [use case #3](#usecase3), you will be able to see a reminder pop-out box like this:   
```

Comment:
I think it would be good to state the assumption that 18 April is within a week from today.

somthing like:

Assuming that the reminder date from USE_CASE_3 is within a week from today, upon the next opening of tinner, you wil be able...